client/dashboard/sites/backup-restore/progress.tsx

```diff
@@ -0,0 +1,70 @@
+import { useQuery } from '@tanstack/react-query';
+import {
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	__experimentalSpacer as Spacer,
+	ProgressBar,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useEffect } from 'react';
+import { siteBackupRestoreProgressQuery } from '../../app/queries/site-backup-restore';
+import Notice from '../../components/notice';
+import noSitesIllustration from '../no-sites-illustration.svg';
+import type { Site } from '../../data/types';
+
+function SiteBackupRestoreProgress( {
+	site,
+	restoreId,
+	onRestoreComplete,
+	onRestoreError,
+}: {
+	site: Site;
+	restoreId: number;
+	onRestoreComplete: () => void;
+	onRestoreError: () => void;
+} ) {
+	const { data: restoreProgress } = useQuery( {
+		...siteBackupRestoreProgressQuery( site.ID, restoreId ),
+		enabled: !! restoreId,
+	} );
+
+	useEffect( () => {
+		if ( restoreProgress?.status === 'finished' ) {
+			onRestoreComplete();
+		} else if ( restoreProgress?.status === 'fail' ) {
+			onRestoreError();
+		}
+	}, [ restoreProgress?.status, onRestoreComplete, onRestoreError ] );
+
+	const isRunning = restoreProgress?.status === 'running';
+
+	return (
+		<>
+			<VStack spacing={ 4 } alignment="center">
+				<img
+					src={ noSitesIllustration }
+					alt=""
+					width={ 408 }
+					height={ 280 }
+					style={ { opacity: 0.2 } }
+				/>
+				<Text size={ 20 }>
+					{ isRunning ? restoreProgress?.message : __( 'Initializing the restore process' ) }
+				</Text>
+				<Text size={ 13 } variant="muted">
+					{ restoreProgress?.percent }% completed
```

Comment:
We should translate it 🙂

Reply:
oops! Addressed it on ddbebc1e31160386d77bb373064af41635cfe229